avm/ptn/dev-ops/cicd-agents-and-runners/tests/e2e/aca-aci/main.test.bicep

```diff
@@ -0,0 +1,82 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults for Azure DevOps self-hosted agents using both Azure Container Instances and Azure Container Apps.'
+metadata description = 'This instance deploys the module with the minimum set of required parameters for Azure DevOps self-hosted agents in Azure Container Instances and Azure Container Apps.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-devopsrunners-${serviceShort}-rg'
+
+#disable-next-line no-hardcoded-location // Due to quotas and capacity challenges, this region must be used in the AVM testing subscription
+var enforcedLocation = 'eastus2'
+
+@description('Required. The name of the Azure DevOps agents pool.')
+param agentsPoolName string = 'agents-pool'
```

Comment:
Sould be coded into the test and not be a parameter

Reply:
Applies also to other tests

Reply:
Resolved

Reply:
Hm... no 😄 Sorry this was just an example.

In general, the test cases should only have parmaeters like
- ResourceGroupName
- NamePrefix
- ServiceShort
- ResourceLocation/EnforcedLocation
- (secrets that you may need to specify, like your PAT that wants to be a parameter)

Everything else should ideally be coded straight into the test invocation (as only then it will also be pulled into the ReadMe examples), or if you need to use the value in more than one location, put it into a variable.

I hope that makes sense :) If not, let's have a chat

Reply:
Done